### ISSUE TITLE
[windows] Tilt sometimes fails syncing with `short read in ReadEvents()`

### ISSUE THREAD

**Issue by joseph-zhong** [tilt-dev/tilt#3353]
Hey, thanks for the quick turnaround for supporting Windows!

I'm running into an issue where Tilt fails to sync without a stacktrace:

![image](https://user-images.githubusercontent.com/4347868/82354132-6b41ab80-99ce-11ea-8b57-6af73686e59d.png)

**Comment by aman-tiwari** [comment]
The results of running `tilt doctor`
```
Tilt: v0.11.4, built 2020-01-31
System: windows-amd64
---
Docker
- Host: [default]
- Version: 1.40
- Builder: 2
Kubernetes
- Env: unknown
- Context: dev
- Cluster Name: <redacted>
- Namespace: <redacted>
- Container Runtime: docker
- Version: v1.14.9-eks-502bfb
---
Thanks for seeing the Tilt Doctor!
Please send the info above when filing bug reports. 💗

The info below helps us understand how you're using Tilt so we can improve,
but is not required to ask for help.
---
Analytics Settings
- User Mode: opt-out
- Machine: [redacted]
- Repo: [redacted]
```

**Comment by joseph-zhong** [comment]
One minor suspicious (possible it's just a typo) is the tilt version, as it should read the recent `v0.13.6` RC build

**Comment by nicks** [comment]
Thanks for the report! Two clarification questions:
1) How often does this happen? 
2) Is it consistently reproducible, or something that just pops up time-to-time unpredictably?

ya, I'd double-check that you're using the right Tilt version, v0.13.6 has major changes to the file-watching code that might have fixed this

**Comment by aman-tiwari** [comment]
Oops, the version mismatch was my bad. I had two versions of tilts on my path, the old v0.11.4 and the new v0.13.6. Trying with 0.13.6 to see if the error happens again.

**Comment by joseph-zhong** [comment]
Closing as it seems to be resolved in 0.13.6
